### PR TITLE
basic explanation in README.md

### PR DESCRIPTION
**Pull request by santiago-rodrig**
# First iteration

Hey guys, I took the time to do the first iteration over the `README.md` file, please read it and let me know if it can be improved (I know it can), until now I think there are some things missing, I'll put them.

- Database configuration
- App configuration
- Models explanation
- Word difficulty ranking algorithm

That's what comes to my mind right now, please notify me somehow if there is actually something else to point out.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,47 @@
...
+This a **vocabulary trainer for foreign languages**, the game is as follows,
+you sign up (just using you email), specify your **original** language and your
+**target** language (the one on that you want to improve your vocabulary)
+pick a *part of speech* category, i.e, pronouns, verbs, etc. After selecting
+the category, click on a level, inside the level you'll need to translate the
+displayed words into your original language, answering correctly gives you
+points! The user with more points will appear at the top of the ranking list of
+the home page. Pretty straightforward, isn't it? Well, that kind of explains
+everything about the app, have a good time while using it! And, if you want to,
+collaborate to its development!
```
Comment (ferreirati) [inline README.md]: Nice intro, very clear and concise. 
I've pasted the text into Grammarly, it finds a couple of fixes. 


image
1. you -> your
2. level -> level,
3. to -> with

File: README.md [hunk 2]
```diff
@@ -1,24 +1,47 @@
...
+### Development environment setup
 
-* Database initialization
+After cloning the repository and changing your current directory
+to the one representing the app, do the following.
 
-* How to run the test suite
+```console
+bundle install && rails db:migrate && rails db:seed
+```
```
Comment (ferreirati) [inline README.md]: On my env I also need to execute `yarn install --check-files`, do u also needed it?

File: README.md (hunk 1 above)
Comment (santiago-rodrig) [reply README.md]: Oh! The next time I'll remember to pass the paragraphs through **grammarly**.

File: README.md (hunk 2 above)
Comment (santiago-rodrig) [reply README.md]: Yes, I forgot it, now should be pointed out.

**Comment by ferreirati** [comment]
Merged. Thanks Santiago.

**Pull request merged**